### DISCUSSION TITLE
2.1.1 crashing on mac 86 (Intel-Catalina) - where can I get another 2.1.0?

### DISCUSSION THREAD

**Discussion by BrentEBeebe** [OrcaSlicer/OrcaSlicer#5939] [General]
2.1.0 was working great. Downloaded/Installed OrcaSlicer_Mac_x86_64_V2.1.1.dmg, now it crashes. Retried this loop several times, no joy.

crash log attached.
[2.1.1 - crash log.txt](https://github.com/user-attachments/files/16043199/2.1.1.-.crash.log.txt)

**Comment by Krookd** [comment]
Same for me. I use OrcaSlicer on an intel mac and V2.1.1 keeps crashing on startup. If you need more information to fix this please tell me

**Comment by LJKraus** [comment]
"- where can I get another 2.1.0? "

Go to https://github.com/SoftFever/OrcaSlicer/releases. Find the version of your choice, open **Assets** near the bottom of the description to download the appropriate package.

**Comment by KatieCakezz** [comment]
Having the same issue on 2.3 latest on silicon Mac m4. immediate force quit.
